code/datums/jobs/job/marines.dm

```diff
@@ -376,12 +376,9 @@ You are also in charge of communicating with command and letting them know about
 		return
 	if(!new_human.assigned_squad)
 		return
-	if(new_human.assigned_squad.squad_leader != new_human)
-		if(new_human.assigned_squad.squad_leader)
-			new_human.assigned_squad.demote_leader()
+	if(!new_human.assigned_squad.squad_leader)
```

Comment:
this no longer demotes an existing ASL? (happy for this to not do that to actual SL's as discussed, but ASL's it definitely should)